kong/plugins/inflate-gzip/handler.lua

```diff
@@ -52,7 +50,16 @@ function plugin:body_filter(conf)
 end
 
 
+function plugin:body_filter(conf)
+  if kong.ctx.plugin.compressed_body then
+    kong.response.set_raw_body(kong.ctx.plugin.compressed_body)
+  end
```

Comment:
```suggestion
  local ctx_plugin = kong.ctx.plugin
  if ctx_plugin.compressed_body then
    kong.response.set_raw_body(ctx_plugin.compressed_body)
    ctx_plugin.compressed_body = nil -- release memory early
  end
```

Release the memory after it is no longer needed, since the ctx tables will be kept around until after the log-phase. We can already drop it here.